packages/lib/services/KeymapService.ts

```diff
@@ -303,6 +303,23 @@ const defaultKeymapItems = {
 		{ accelerator: 'Cmd+P', command: 'gotoAnything' },
 		{ accelerator: 'Shift+Cmd+P', command: 'commandPalette' },
 		{ accelerator: 'F1', command: 'help' },
+		{ accelerator: 'Cmd+D', command: 'editor.deleteLine' },
+		{ accelerator: 'Cmd+Z', command: 'editor.undo' },
+		{ accelerator: 'Cmd+Y', command: 'editor.redo' },
+		{ accelerator: 'Cmd+Home', command: 'editor.goDocStart' },
+		{ accelerator: 'Cmd+End', command: 'editor.goDocEnd' },
+		{ accelerator: 'Cmd+Left', command: 'editor.goGroupLeft' },
+		{ accelerator: 'Cmd+Right', command: 'editor.goGroupRight' },
+		{ accelerator: 'Alt+Left', command: 'editor.goLineStart' },
+		{ accelerator: 'Alt+Right', command: 'editor.goLineEnd' },
+		{ accelerator: 'Alt+Backspace', command: 'editor.delGroupBefore' },
+		{ accelerator: 'Alt+Delete', command: 'editor.delGroupAfter' },
+		{ accelerator: 'Cmd+[', command: 'editor.indentLess' },
+		{ accelerator: 'Cmd+]', command: 'editor.indentMore' },
+		{ accelerator: 'Cmd+/', command: 'editor.toggleComment' },
+		{ accelerator: 'Option+Cmd+A', command: 'editor.sortSelectedLines' },
+		{ accelerator: 'Option+Up', command: 'editor.swapLineUp' },
+		{ accelerator: 'Option+Down', command: 'editor.swapLineDown' },
```

Comment:
I though we established that `Option` does not work and has to be replaced with `Alt`.

Reply:
Oh, these are the ones in the Keyboard Editor. Sorry, I missed that.

Reply:
It doesn't work when binding to the codemirror editor, but it works for the regular menu items (I think). When commands get registered with the CodeMirror instance there is a regex translation layer that changes Option -> Alt

Reply:
No worries, you comment made me realize that I was translate Opt -> Alt, not Option. It's fixed now!

Reply:
Right, it also has to change the + to - I think.